plugin/ctrlsf.vim

```diff
@@ -219,21 +219,21 @@ com! -n=0                                         CtrlSFToggle  call ctrlsf#Togg
 " }}}
 
 " Maps {{{1
-nnoremap        <Plug>CtrlSFPrompt    :CtrlSF<Space>
-nnoremap <expr> <Plug>CtrlSFCwordPath <SID>SearchCwordCmd('CtrlSF', 0)
-nnoremap <expr> <Plug>CtrlSFCwordExec <SID>SearchCwordCmd('CtrlSF', 1)
-vnoremap <expr> <Plug>CtrlSFVwordPath <SID>SearchVwordCmd('CtrlSF', 0)
-vnoremap <expr> <Plug>CtrlSFVwordExec <SID>SearchVwordCmd('CtrlSF', 1)
-nnoremap <expr> <Plug>CtrlSFPwordPath <SID>SearchPwordCmd('CtrlSF', 0)
-nnoremap <expr> <Plug>CtrlSFPwordExec <SID>SearchPwordCmd('CtrlSF', 1)
-
-nnoremap        <Plug>CtrlSFQuickfixPrompt :CtrlSFQuickfix<Space>
-nnoremap <expr> <Plug>CtrlSFQuickfixCwordPath <SID>SearchCwordCmd('CtrlSFQuickfix', 0)
-nnoremap <expr> <Plug>CtrlSFQuickfixCwordExec <SID>SearchCwordCmd('CtrlSFQuickfix', 1)
-vnoremap <expr> <Plug>CtrlSFQuickfixVwordPath <SID>SearchVwordCmd('CtrlSFQuickfix', 0)
-vnoremap <expr> <Plug>CtrlSFQuickfixVwordExec <SID>SearchVwordCmd('CtrlSFQuickfix', 1)
-nnoremap <expr> <Plug>CtrlSFQuickfixPwordPath <SID>SearchPwordCmd('CtrlSFQuickfix', 0)
-nnoremap <expr> <Plug>CtrlSFQuickfixPwordExec <SID>SearchPwordCmd('CtrlSFQuickfix', 1)
+nnoremap <silent> <Plug>CtrlSFPrompt    :CtrlSF<Space>
```

Comment:
Appending `<silent>` does not make sense for `<Plug>CtrlSFPrompt`, this map is used for waiting your further input, `<silent>` makes nothing visible for user and can be confusing.